cmd/ferretdb/main.go

```diff
@@ -104,7 +104,7 @@ var pgFlags struct {
 //
 // See main_sqlite.go.
 var sqliteFlags struct {
-	SQLiteURI string `name:"sqlite-uri" default:"." help:"Directory path or 'file' URI for 'sqlite' handler."`
+	SQLiteURI string `default:"." help:"'file' scheme URI with directory path for 'sqlite' handler." name:"sqlite-uri" `
```

Comment:
I changed the order because of that:
```
Error: cmd/ferretdb/main.go:107:19: tag is not aligned , should be: default:"file://." help:"'file' scheme URI with directory path for 'sqlite' handler." name:"sqlite-uri" (tagalign)
	SQLiteURI string `name:"sqlite-uri" default:"file://." help:"'file' scheme URI with directory path for 'sqlite' handler." `
```
Should I add a `nolint` here?

Reply:
https://golangci-lint.run/usage/linters/#tagalign